tests/test_quality.py

```diff
@@ -266,3 +269,12 @@ def test_quality_multi_process(nlp):
     docs = nlp.pipe(texts, n_process=2)
     for doc in docs:
         assert doc._.quality
+
+@pytest.mark.parametrize("vocab", [None, {"This", "is", "a", "test"}])
+def test_oov_ratio(vocab):
+    """Test the oov_ratio function."""
+    nlp = spacy.load("en_core_web_md")
+    doc = nlp("This is a test")
+    assert oov_ratio(doc, vocab) == 0
+    doc = nlp("This is a nonwrod")
+    assert oov_ratio(doc, vocab) == 0.25
```

Comment:
Why parametrize? the test can't handle a new vocab anyway

```suggestion
def test_oov_ratio():
    vocab = {"This", "is", "a", "test"}
    nlp = spacy.load("en_core_web_md")
    doc = nlp("This is a test")
    assert oov_ratio(doc, vocab) == 0
    doc = nlp("This is a nonwrod")
    assert oov_ratio(doc, vocab) == 0.25
```

Reply:
Parameterized to ensure that we get the same results with the spacy model's vocab (None) and with a specified vocab

Reply:
Ahh right I see. So then I would rewrite the test to take in text,oov_ratio,vocab so it will become 4 examples.

Reply:
(not resolved)